arch/Makefile.rules

```diff
@@ -0,0 +1,14 @@
+_UKARCH_TLS_HAVE_TCB=
+
+## Make sure that the reserved space for a TCB is set only once.
+## A second call of this function will make compiling stop.
+## Usage:
+##  $(eval $(call ukarch_tls_tcb_reserve,<tcb_size_in_bytes>))
```

Comment:
Any reason to use double comment signs (`##`) instead of a single one? Is this something to insist on?

Reply:
Good question. It seems that I took it over from uknetdev but it is obviously not needed. I'll make it to single `#`.